internal/db/changelog.go

```diff
@@ -1,38 +1,60 @@
 package db
 
 import (
-	"log"
+	"github.com/pkg/errors"
+
 	// Blank import required for mysql driver
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/jimmyjames85/bouncecm/internal/models"
 )
 
 // Changelog - Function to pull all rules from db
-func  (c *Client) Changelog() (*models.ChangelogTable, error) {
+func  (c *Client) GetAllChangelogEntries() (*[]models.ChangelogEntry, error) {

```

Comment:
This should return `([]models.ChangelogEntry, error)`. In go slices are passed by pointer, so no need for a pointer to a slice